RELEASE/scripts/autoscend.ash

```diff
@@ -6977,7 +6979,13 @@ boolean L11_mauriceSpookyraven()
 			buffMaintain($effect[Sweetbreads Flamb&eacute;], 0, 1, 1);
 		}
 
-		addToMaximize("100ml 82max");
+		// Maximize Asdon usage
+		if(monster_level_adjustment() <= 81)
+		{
+			asdonBuff($effect[Driving Recklessly]);
+		}
+
+		addToMaximize("500ml 82max");
```

Comment:
im not great with maximizer strings, whats the point of having 500ml is we only want a max of 82? Or maybe i dont understand exactly what that means.

Reply:
The 500 is the weight given to every point of +ML, up to a maximum of 82 +ML points.

This the high weight helps prevent the +ML Form getting overridden by other maximizer strings that are added during normal adventuring.